libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -531,26 +548,26 @@ void AP_InertialSensor_Backend::update_gyro(uint8_t instance)
     }
 
     // possily update the harmonic notch filter parameters
-    if (!is_equal(_last_harmonic_notch_bandwidth_hz, gyro_harmonic_notch_bandwidth_hz()) ||
-        !is_equal(_last_harmonic_notch_attenuation_dB, gyro_harmonic_notch_attenuation_dB()) ||
+    if (!is_equal(_last_harmonic_notch_bandwidth_hz, harmonic_notch_bandwidth_hz()) ||
+        !is_equal(_last_harmonic_notch_attenuation_dB, harmonic_notch_attenuation_dB()) ||
         sensors_converging()) {
-        _imu._gyro_harmonic_notch_filter[instance].init(_gyro_raw_sample_rate(instance), gyro_harmonic_notch_center_freq_hz(), gyro_harmonic_notch_bandwidth_hz(), gyro_harmonic_notch_attenuation_dB());
-        _last_harmonic_notch_center_freq_hz = gyro_harmonic_notch_center_freq_hz();
-        _last_harmonic_notch_bandwidth_hz = gyro_harmonic_notch_bandwidth_hz();
-        _last_harmonic_notch_attenuation_dB = gyro_harmonic_notch_attenuation_dB();
-    } else if (!is_equal(_last_harmonic_notch_center_freq_hz, gyro_harmonic_notch_center_freq_hz())) {
-        _imu._gyro_harmonic_notch_filter[instance].update(gyro_harmonic_notch_center_freq_hz());
-        _last_harmonic_notch_center_freq_hz = gyro_harmonic_notch_center_freq_hz();
+        _imu._gyro_harmonic_notch_filter[instance].init(_gyro_raw_sample_rate(instance), harmonic_notch_center_freq_hz(), harmonic_notch_bandwidth_hz(), harmonic_notch_attenuation_dB());
+        _last_harmonic_notch_center_freq_hz = harmonic_notch_center_freq_hz();
+        _last_harmonic_notch_bandwidth_hz = harmonic_notch_bandwidth_hz();
+        _last_harmonic_notch_attenuation_dB = harmonic_notch_attenuation_dB();
+    } else if (!is_equal(_last_harmonic_notch_center_freq_hz, harmonic_notch_center_freq_hz())) {
+        _imu._gyro_harmonic_notch_filter[instance].update(harmonic_notch_center_freq_hz());
+        _last_harmonic_notch_center_freq_hz = harmonic_notch_center_freq_hz();
     }
     // possily update the notch filter parameters
-    if (!is_equal(_last_notch_center_freq_hz, _gyro_notch_center_freq_hz()) ||
-        !is_equal(_last_notch_bandwidth_hz, _gyro_notch_bandwidth_hz()) ||
-        !is_equal(_last_notch_attenuation_dB, _gyro_notch_attenuation_dB()) ||
+    if (!is_equal(_last_gyro_notch_center_freq_hz, _notch_center_freq_hz()) ||
```

Comment:
I know @rmackay9 objects to functions starting with _ so if you are going to change the names probably remove the _.

Reply:
Sure